### ISSUE TITLE
openassistant.org

### ISSUE THREAD

**Issue by testman42** [vavrek/openassistant#11]
http://openassistant.org is ded.
Press F to pay respects.

<kbd>[F](http://knowyourmeme.com/memes/press-f-to-pay-respects)</kbd>

**Comment by Whytehorse** [comment]
Yes it is, can we recover all the data we put into the forums and wiki there?

**Comment by testman42** [comment]
The site is back, woo :)

![Show me what you got](http://openassistant.org/wp-content/uploads/2016/10/oa_vr_range1-1024x576.png)
[Show me what you got](http://i.imgur.com/7toTb0T.png)
